cachito/workers/tasks/general.py

```diff
@@ -95,3 +98,38 @@ def failed_request_callback(context, exc, traceback, request_id):
         msg = 'An unknown error occurred'
 
     set_request_state(request_id, 'failed', msg)
+
+
+@app.task
+def create_bundle_archive(app_archive_path, request_id):
```

Comment:
Don't we need to change `resolve_gomod_deps` to return the `app_archive_path` for this to work? It currently returns the list of deps.

Reply:
The Celery task is `fetch_gomod_source` which returns `app_archive_path`.  `fetch_gomod_source` wraps `resolve_gomod_deps`.